corehq/apps/app_manager/util.py

```diff
@@ -540,7 +541,13 @@ def _app_callout_templates():
         yield data
 
 
+@memoized
```

Comment:
Memoized is only an in-memory cache, so it isn't the thing to use here, you'll want to cache across requests. I was thinking you could quickcache, probably for a while, maybe a week or longer. Check out [these docs](https://commcare-hq.readthedocs.io/caching_and_memoization.html) if you haven't seen them.

Reply:
> Memoized is only an in-memory cache

Ah, I thought it was the other way around (but haven't read that docs to be honest). Probably the "quick" in `quickcache` that got my mind on the wrong track :shrug

Reply:
https://github.com/dimagi/commcare-hq/pull/35628/commits/6968738f7f50ba9f98bb1e8f9cf7f64606ec7193

Reply:
cool, thanks for updating